src/replit_river/client.py

```diff
@@ -129,7 +128,7 @@ async def send_subscription(
         request_serializer: Callable[[RequestType], Any],
         response_deserializer: Callable[[Any], ResponseType],
         error_deserializer: Callable[[Any], Any],
-    ) -> AsyncGenerator[Union[ResponseType, ErrorType], None]:
+    ) -> AsyncGenerator[Union[ResponseType, RiverError], None]:
```

Comment:
this removes the variance from the result type here, right? This seems like it would make the error types here squash down to `RiverError` instead of "some type that satisfies `RiverError`"

Reply:
I got a warning here that ErrorType should -> RiverError here after I changed the return type of error_deserializer to Any.